src/components/common/ViewerToolbar.jsx

```diff
@@ -55,35 +55,52 @@ export default class ViewerToolbar extends Component {
 
         let levels = false;
         let sliders = _.times(3, (i) => {
-            let val = customScale[i] ? customScale[i] : featuresScale[i];
+            let val = customScale[i] ? customScale[i] : [0, featuresScale[i]];
             if (
                 activeFeatures[i] &&
                 activeFeatures[i].feature.length &&
                 featuresScale[i]
             ) {
                 levels = true;
+                let midScalePerc =
+                    (1 -
+                        (customScale[i][1] + customScale[i][0]) /
+                            2 /
+                            featuresScale[i]) *
+                    100;
                 return (
-                    <TooltipSlider
+                    <TooltipRange
                         vertical
                         key={i}
                         style={{
                             minHeight: '400px',
                             margin: '5px',
                             float: 'left',
                         }}
-                        trackStyle={{
+                        trackStyle={[
+                            {
+                                background:
+                                    'linear-gradient(' + colors[i] + ', black)',
+                            },
+                        ]}
+                        handleStyle={[{ border: '2px solid ' + colors[i] }]}
+                        railStyle={{
                             background:
-                                'linear-gradient(to top, black, ' +
+                                'linear-gradient(' +
                                 colors[i] +
-                                ')',
+                                ' ' +
+                                midScalePerc +
+                                '%, black ' +
+                                midScalePerc +
+                                '%)',
```

Comment:
Also consider making this a template literal with `midScale * 100` instead of `midScalePerc`.